app/components/Home.tsx

```diff
@@ -31,20 +35,22 @@ const MainBrowserWindow: React.FunctionComponent<Props> = (props: Props) => {
     updateTabs(modifiedTabs);
     setTabIndex(modifiedTabs.length - 1);
     updateUrl('');
+    setModal(modal);
   }
 
+  const onTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => updateTitle(event.target.value)
   const onUrlChange = (event: React.ChangeEvent<HTMLInputElement>) => updateUrl(event.target.value)
 
-  const onRemovingTab = index => {
-    const tabItem = tabs.filter((tab,tabIndex) => tabIndex !== index)
+  const onRemovingTab = (index: number) => {
+    const tabItem = tabs.filter((_,tabIndex) => tabIndex !== index)
```

Comment:
No types defined for
1. `onRemovingTab` return value
2. `tabItem` and `tabeIndex` type